### DISCUSSION TITLE
Captions and alt text for cover images

### DISCUSSION THREAD

**Discussion by jackbaty** [jpanther/congo#381] [Ideas]
Have there been any considerations around including a caption for cover/featured images? Same for alt text. I'm trying to be better with accessibility and alt-text for images seems important. I suppose it would need to be via front matter?

**Comment by jpanther** [comment]
Thanks for the suggestion. This is definitely something that I could look to add to the theme as Congo is built to be highly accessible. Front matter is the most likely scenario for providing this information as it's not something that could easily be placed anywhere else without making it complicated.

**Comment by HeyItsGilbert** [comment]
I noticed a few images didn't have an alt text (discovered via https://pagespeed.web.dev/). I suspect we could default to summary. If you define coverAlt or featureAlt it would use those? I'm happy to attempt a PR.

**Comment by jpanther** [comment]
I'm happy to report that this feature has been implemented and will be included in the next theme release.

There are new front matter parameters for specifying the image filenames as well as the alt text and cover image captions.
